rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -2172,6 +2172,51 @@ namespace vk
 			src.first = false;
 			return VK_SUCCESS;
 		}
+#else
+
+	class swapchain_generic : public native_swapchain_base
+	{
+
+	public:
+		swapchain_generic(physical_device &gpu, uint32_t _present_queue, uint32_t _graphics_queue, VkFormat format = VK_FORMAT_B8G8R8A8_UNORM)
+		: native_swapchain_base(gpu, _present_queue, _graphics_queue, format)
+		{}
+
+		~swapchain_generic(){}
+
+		bool init() override
+		{
+			//TODO: get from `nsView`
```

Comment:
Don't bother implementing anything here since its unlikely anyone is going to implement this fallback for wayland. Stub everything with fmt::throw.